lib/howdy.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.greeting = function(name) {
+  if (arguments.length === 0) throw new Error('no name given');
+  console.log(`What's up, ${name}?!`);
+  return `hello ${name}`;
+};
+
+exports.multipleGreetings = function(termInput) {
```

Comment:
Phenomenal logic here. :) Really a fan of being able to handle multiple inputs like this!